### ISSUE TITLE
Add docs for 24/7 usage

### ISSUE THREAD

**Issue by brandonmcconnell** [ZekeSnider/Jared#66]
Does the mac this script runs on need to be on and unlocked in order for the script to run? Is there any workaround for this, or any macOS server that can run us around the clock?

I’d like to enable this for several chats, but that would likely require a server, which may be a showstopper since iMessage doesn’t have an open API (AFAIK).

**Comment by ZekeSnider** [comment]
Yes the mac needs to be on and running, but not necessarily unlocked. You can use a tool like [Amphetamine](https://apps.apple.com/us/app/amphetamine/id937984704?mt=12) to accomplish this.

And correct that iMessage does not have a public API that accomplishes this. There is iMessage business chat and iMessage apps, neither of which serve this use case specifically.

**Comment by jjroley** [comment]
@brandonmcconnell have you found a good solution for this? I tried Amphetimene but the messages still don’t send automatically when the Mac is sleeping. Do you just have to leave the display on at all times?

**Comment by brandonmcconnell** [comment]
Yeah, that appears to be the case, unfortunately. Hopefully, @Apple just gives us an iMessage API we can rely on instead of this soon.

**Comment by jjroley** [comment]
I was able to get Jared to continue running, in preferences you can disable the computer from sleeping while the display is off. No need for even Amphetamine.
